### DISCUSSION TITLE
LoggingEventCompositeJsonEncoder with Spring Boot 3

### DISCUSSION THREAD

**Discussion by KulkarD1** [logfellow/logstash-logback-encoder#904] [Q&A] [answered]
Has anyone used `net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder` with Spring Boot 3? No logs are generated with Boot v3.0.0-RC2 while the exact same code/ configuration works great with Boot v2.7.5

I have a logback-spring.xml with-
```
    <appender name="JsonConsole" class="ch.qos.logback.core.ConsoleAppender">
        <encoder class="net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder">
```

Since there are no logs, i am unable to see any exceptions that would help with troubleshooting. Any clues on how to troubleshoot such an issue would be great as well. Thanks!

**Comment by brenuart** [comment]
Are you getting any output whe the ConsoleAppender is configured with a regular Logback PatternLayout ?

**Reply by KulkarD1** [reply]
yes log output is fine with this- 

```
<appender name="Console" class="ch.qos.logback.core.ConsoleAppender">
    <encoder>
        <pattern>${CONSOLE_LOG_PATTERN}</pattern>
        <charset>utf8</charset>
    </encoder>
</appender>
```

**Comment by KulkarD1** [comment, marked as answer]
Json Logging is working fine with the Boot v3.0.0 GA that was just released, maybe it was something specific to RC2. I will try and figure out why. But this can be closed. Thanks!
